_posts/2022/2022-10-31-use-zero-for-integer-float-comparison.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Use zero? for comparison of numerics like Integer, Float and BigDecimal"
+description: "Other languages don't have this"
+layout: article
+category: ruby
+image:
+  base: "2022/use-zero-for-integer-float-comparison"
+  alt: "A zero on a sign on the wall"
+  credit: "Bernard Hermant"
+  source: "https://unsplash.com/photos/rfpSOlH1JlQ"
+---
+
+Ruby often takes a more “readable” approach to its syntax and standard library of included methods than other languages.
+
+An example of this is the common programming comparison of numeric variable to zero.
+
+See the documentation of the `.zero?` method on [`Integer`](https://ruby-doc.org/core-3.1.2/Integer.html#zero-3F-method), [`Float`](https://ruby-doc.org/core-3.1.2/Float.html#zero-3F-method), [`Numeric`](https://ruby-doc.org/core-3.1.2/Numeric.html#zero-3F-method) and [`BigDecimal`](https://ruby-doc.org/stdlib-3.1.2/libdoc/bigdecimal/rdoc/BigDecimal.html#zero-3F-method).
```

Comment:
Hmm, at this point, not clear on where this article is going.

Reply:
IT'S A JOURNEY.

A short one.